lib/galaxy/webapps/galaxy/config_schema.yml

```diff
@@ -274,13 +290,14 @@ mapping:
 
       conda_prefix:
         type: str
-        default: <tool_dependency_dir>/_conda
         required: false
         desc: |
-          conda_prefix is the location on the filesystem where Conda packages and environments are installed
-          IMPORTANT: Due to a current limitation in conda, the total length of the
+          conda_prefix is the location on the filesystem where Conda packages and environments are
+          installed IMPORTANT: Due to a current limitation in conda, the total length of the
```

Comment:
Pretty sure this limitation hasn't been true since everything was rebuilt with conda-build 2.0 a couple years ago, @bgruening? Not that you added this but while you're modifying it, might as well correct it.

Reply:
Yes I think we can remove it. if there are still old packages around with this limitation we should rebuild them on conda.

Reply:
Thanks for the confirmation!

Reply:
Done. Removed starting with "IMPORTANT..."